tutorials/01_least_squares_optimization.ipynb

```diff
@@ -173,37 +163,20 @@
      "output_type": "stream",
      "text": [
       "Nonlinear optimizer. Iteration: 0. Error: 38.42743682861328\n",
-      "Nonlinear optimizer. Iteration: 1. Error: 9.609884262084961\n",
-      "Nonlinear optimizer. Iteration: 2. Error: 2.405491828918457\n",
-      "Nonlinear optimizer. Iteration: 3. Error: 0.6043925285339355\n",
-      "Nonlinear optimizer. Iteration: 4. Error: 0.15411755442619324\n",
-      "Nonlinear optimizer. Iteration: 5. Error: 0.04154873266816139\n",
-      "Nonlinear optimizer. Iteration: 6. Error: 0.013406438753008842\n",
-      "Nonlinear optimizer. Iteration: 7. Error: 0.006370890885591507\n",
-      "Nonlinear optimizer. Iteration: 8. Error: 0.0046120136976242065\n",
-      "Nonlinear optimizer. Iteration: 9. Error: 0.0041722883470356464\n",
-      "Nonlinear optimizer. Iteration: 10. Error: 0.004062363877892494\n",
-      "Nonlinear optimizer. Iteration: 11. Error: 0.004034876357764006\n",
-      "Nonlinear optimizer. Iteration: 12. Error: 0.004028005059808493\n",
-      "Nonlinear optimizer. Iteration: 13. Error: 0.004026289563626051\n",
-      "Nonlinear optimizer. Iteration: 14. Error: 0.004025860223919153\n",
-      "Nonlinear optimizer. Iteration: 15. Error: 0.00402575358748436\n",
-      "Best solution: {'a': tensor([[0.9945]]), 'b': tensor([[0.5018]])}\n"
+      "Best solution: {'a': tensor([[2.]]), 'b': tensor([[1.]])}\n"
      ]
     },
     {
-     "data": {
-      "text/plain": [
-       "Text(0, 0.5, 'y')"
-      ]
-     },
-     "execution_count": 4,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/mukadam/opt/anaconda3/envs/theseus/lib/python3.7/site-packages/theseus/optimizer/nonlinear/nonlinear_optimizer.py:191: RuntimeWarning: There was an error while running the linear optimizer. Original error message: cholesky_cpu: For batch 0: U(1,1) is zero, singular U..\n",
```

Comment:
There seems to be a weird behavior when running T1 with python 3.7 where the optimization is singular. This goes away when run on python 3.8.
cc @luisenp @vshobha